<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Frame</class>
 <widget class="QWidget" name="Main_Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resorces/resorces/chatRoom.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_user_portrait">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/resorces/1.bmp);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_user_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b_21</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>捣蛋1号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>85</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>261</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_friend">
    <attribute name="icon">
     <iconset>
      <normaloff>:/image/image/menu_friend_list.PNG</normaloff>:/image/image/menu_friend_list.PNG</iconset>
    </attribute>
    <attribute name="title">
     <string>好友</string>
    </attribute>
    <widget class="QTreeView" name="treeView_friends">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>-1</y>
       <width>261</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>:/image/image/menu_group_list.PNG</normaloff>:/image/image/menu_group_list.PNG</iconset>
    </attribute>
    <attribute name="title">
     <string>群</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolButton" name="button_add_friend">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_user_signature">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
